libvast/builtins/formats/zeek_tsv.cpp

```diff
@@ -0,0 +1,757 @@
+            co_yield {};
+            continue;
+          }
+          if (line->starts_with("#separator")) {
+            if (not closed) {
+              ctrl.abort(caf::make_error(
+                ec::syntax_error, fmt::format("Parsing Zeek TSV failed: "
+                                              "previous logs are still open")));
+              co_return;
+            }
+            closed = false;
+            auto finished = b.finish();
+            if (metadata.output_slice_schema)
+              finished
+                = cast(std::move(finished), metadata.output_slice_schema);
+            co_yield std::move(finished);
+            auto parsed = metadata.parse_header(it, lines, ctrl);
+            if (not parsed) {
+              ctrl.abort(parsed.error());
+              co_return;
+            }
+            xs.resize(metadata.fields.size());
+            b = table_slice_builder{metadata.temp_slice_schema};
+            ++it;
+          }
+          if (closed) {
+            ctrl.abort(caf::make_error(ec::syntax_error,
+                                       fmt::format("Parsing Zeek TSV failed: "
+                                                   "additional data after "
+                                                   "#close should be "
+                                                   "preceded by Zeek header")));
+            co_return;
+          }
+          auto values = detail::split((*it).value(), metadata.sep);
+          if (values.size() != metadata.fields.size()) {
+            ctrl.warn(caf::make_error(
+              ec::parse_error,
+              fmt::format("Zeek TSV parser skipped line: "
+                          "expected {} fields but got "
+                          "{}",
+                          metadata.fields.size(), values.size())));
+            continue;
+          }
+          for (auto i = size_t{0}; i < values.size(); ++i) {
+            auto value = values[i];
+            if (metadata.is_unset(value)) {
+              xs[i] = caf::none;
+            } else if (metadata.is_empty(value)) {
+              xs[i] = caf::get<record_type>(metadata.temp_slice_schema)
+                        .field(i)
+                        .type.construct();
+            } else if (!metadata.parsers[i](values[i], xs[i])) {
+              ctrl.warn(caf::make_error(ec::parse_error,
+                                        fmt::format("Zeek TSV parser skipped "
+                                                    "line: failed"
+                                                    "to parse value '{}'",
+                                                    value)));
+              continue;
+            }
+          }
+          for (auto i = size_t{0}; i < xs.size(); ++i) {
+            auto&& x = xs[i];
+            if (not b.add(make_data_view(x))) {
+              ctrl.abort(caf::make_error(ec::parse_error,
+                                         fmt::format("Zeek TSV parser failed "
```

Comment:
```suggestion
                                         fmt::format("zeek-tsv parser failed "
```

Pick anything as prefix (I went with `zeek-tsv`), but use it consistently.